qiita_pet/handlers/websocket_handlers.py

```diff
@@ -74,7 +74,24 @@ def on_close(self):
         self.redis.disconnect()
 
 
+class SelectedSocketHandler(WebSocketHandler, BaseHandler):
+    """Websocket for removing samples on default analysis display page"""
+    @authenticated
+    def on_message(self, msg):
+        # When the websocket receives a message from the javascript client,
+        # parse into JSON
+        msginfo = loads(msg)
+        default = Analysis(self.current_user.default_analysis)
+        if 'samples' not in msginfo:
```

Comment:
if the object sent over the line always includes `samples` then the message is consistent. In this branch, the message would just need to have `samples` associated with `[]`